examples/create-react-app-with-flow/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "create-react-app-with-flow",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "jss": "^8.1.0",
+    "jss-preset-default": "^3.0.0",
+    "material-ui": "1.0.0-beta.12",
+    "prop-types": "^15.6.0",
+    "react": "^16.0.0",
```

Comment:
can we use "latest"?

Reply:
We cannot if we want the `from scratch` to work.  `flow-typed install` will fail.